src/style.css

```diff
@@ -1,3 +1,66 @@
+@font-face {
+  font-family: Roboto;
+  src: url(./font/Roboto-Medium.ttf);
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  width: 100%;
+  align-items: center;
+}
+
+.nav {
+  height: 60px;
+  text-align: center;
+  line-height: 60px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
```

Comment:
redundant
1. it's not a good practice to hardcode height for elements with content inside. you can apply padding to a link to make height
2. font styles you'd better add to a link as well 
```suggestion
```